ELECTIONS.md

```diff
@@ -496,4 +496,4 @@ Eiffel TC [agreed](https://github.com/eiffel-community/community/blob/master/mee
 
 ### Eligible Community Members
 
-Eiffel TC [agreed](https://github.com/eiffel-community/community/blob/master/meetings/MEETINGS_BTC.md#agenda-and-notes-1) to appoint Co Chairs within from the elected TC without an election.
\ No newline at end of file
+Eiffel TC [agreed](https://github.com/eiffel-community/community/blob/master/meetings/MEETINGS_BTC.md#agenda-and-notes-1) to appoint Co Chairs within from the elected TC without an election.
```

Comment:
This hunk is because vim always makes sure files end with a newline character (as they should).